src/index.html

```diff
@@ -5,8 +5,343 @@
+          </li>
+              
+          <li class="nav__item">
+            <a class="nav__link" href="#gadgets">Gadgets</a>
+          </li>
+  
+          <li class="nav__item">
+            <a class="nav__link" href="#tablets">Tablets</a>
+          </li>
+  
+          <li class="nav__item">
+            <a class="nav__link" href="#photo">Photo</a>
+          </li>
+            
+          <li class="nav__item">
+            <a class="nav__link" href="#video">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main">
+      <div class="catalog">
+        <div class="card catalog__card" data-qa="card">
+          <div class="card__item">
+            <img 
+              src="images/imac.jpeg" 
+              alt="screen Product"
+              class="card__img"
+            >
+          </div>
+          <div class="card__description">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</div>
+          <div class="card__code">Product code: 195434</div>
+          <div class="card__stars">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <div class = "card__review">Reviews: 5</div>
+          </div>
+          <div class="card__price">
+            <div class="card__price-text">Price:</div>
+            <div class="card__price-text-value">$2,199</div>
+          </div>
+        
+          <div class="card__button">
+            <a 
+              href="#"
+              class="card-link"
+              data-qa="card-hover"
+            >
+              Buy
+            </a>
+          </div>
+        </div>
+      
+
+        <div class="card catalog__card">
+          <div class="card__item">
+            <img 
+              src="images/imac.jpeg" 
+              alt="screen Product"
+              class="card__img"
+            >
+          </div>
+          <div class="card__description">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</div>
+          <div class="card__code">Product code: 195434</div>
+          <div class="card__stars">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <div class = "card__review">Reviews: 5</div>
+          </div>
+          <div class="card__price">
+            <div class="card__price-text">Price:</div>
+            <div class="card__price-text-value">$2,199</div>
+          </div>
+        
+          <div class="card__button">
+            <a 
+              href="#"
+              class="card-link"
+           
+            >
```

Comment:
fix other places too
```suggestion
              class="card-link"
            >
```